test/correctness/simd_op_check.cpp

```diff
@@ -41,6 +41,26 @@ int my_process_id = 0;
 // width and height of test images
 const int W = 256*3, H = 100;
 
+bool can_run_code() {
+
+  // If we can (target matches host), run the error checking Func.
```

Comment:
The other thing I've noticed frequently in Hexagon code is that we always use braces, even for control flow with single statement bodies.

We have a clang-format file at the root of the repo that might be useful for this. I don't think I've actually seen an official coding convention myself, I just try to mirror what I see in existing code :)